okex-client/src/client/mod.rs

```diff
@@ -67,10 +67,49 @@ pub struct PositionId {
     pub value: String,
 }
 
+#[derive(Debug, Clone)]
+pub enum OkexInstrumentId {
+    BtcUsdSwap,
+    BtcUsd,
+}
+
+#[derive(Debug, Clone)]
+pub enum OkexMarginMode {
+    Cross,
+    Isolated,
+}
+
+#[derive(Debug, Clone)]
+pub enum OkexPositionSide {
+    LongShort(String),
+    Net(String),
+}
+
+impl OkexPositionSide {
+    pub fn long() -> Self {
+        Self::LongShort("long".to_string())
+    }
+
+    pub fn short() -> Self {
+        Self::LongShort("short".to_string())
+    }
+
+    pub fn net() -> Self {
+        Self::Net("net".to_string())
+    }
+}
+
+#[derive(Debug, Clone)]
+pub enum OkexOrderSide {
+    Buy,
+    Sell,
+}
+
 pub struct OkexClientConfig {
     pub api_key: String,
     pub passphrase: String,
     pub secret_key: String,
+    pub simulated: String,
```

Comment:
shouldn't this be a boolean? How do callers know what values are accepted here?

Reply:
Fixed